src/main/java/com/faforever/client/fx/JavaFxUtil.java

```diff
@@ -454,18 +462,22 @@ public static void bindTextFieldAndRangeSlider(TextField lowValueTextField, Text
         @Override
         public String toString(Number number) {
           if (!number.equals(value)) {
-            return String.valueOf(number.intValue());
+            return format.format(number);
           } else {
             return "";
           }
         }
 
         @Override
         public Number fromString(String string) {
-          if (NumberUtils.isParsable(string)) {
-            return Double.parseDouble(string);
-          } else {
-            if (!string.equals("-") && !string.equals(".")) {
+          String decimalSeparator = Character.toString(format.getDecimalFormatSymbols().getDecimalSeparator());
+          try {
+            Number number = format.parse(string);
+            String decimalSeparatorSuffix = string.endsWith(decimalSeparator) && format.getMaximumFractionDigits() > 0 ? decimalSeparator : "";
+            textField.setText(format.format(number) + decimalSeparatorSuffix);
+            return number;
+          } catch (ParseException e) {
+            if (!string.equals("-") && !string.equals(decimalSeparator)) {
```

Comment:
Is this all for locale handling? I though Decimal format could handle different locales

Reply:
it was there before but hard-coded to US locale (dot as decimal separator). not sure if the main purpose was to be able to set for example 0.8 as `.8` but I left this as a possibility even in other locales